src/js/components/MarathonTaskDetailsList.js

```diff
@@ -2,53 +2,29 @@ import React from 'react';
 
 import DescriptionList from './DescriptionList';
 import MarathonStore from '../stores/MarathonStore';
+import TaskUtil from '../utils/TaskUtil';
 
 class MarathonTaskDetailsList extends React.Component {
   getTaskEndpoints(task) {
-    if ((task.ports == null || task.ports.length === 0) &&
-        (task.ipAddresses == null || task.ipAddresses.length === 0)) {
-      return 'None';
+    let endpoints = TaskUtil.getTaskEndpoints(task);
+
+    let endpointURLs = endpoints.hosts.reduce(function (memo, host) {
+      return memo.concat(endpoints.ports.map(function (port) {
+        return `${host}:${port}`;
+      }));
+    }, []);
```

Comment:
That might be a good idea, but probably for a separate PR? Or perhaps even a component. I'm just put up PR for a component which renders the endpoint links using `TaskUtil#getTaskEndpoints` in a specific way.